common/hyperscaler/rules/model/config_test.go

```diff
@@ -0,0 +1,43 @@
+package model
+
+import (
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestLoad(t *testing.T) {
+	t.Run("should load rules from a file", func(t *testing.T) {
```

Comment:
In this test case we test libraries (reading file from os and reading yaml).

Reply:
So basically we create slice of strings:
```yaml
rule:
  - rule1
  - rule2
 ```
write it to the file, read it from the file, unmarshal yaml so we get []{"rule1", "rule2"}

Reply:
What kind of flaw can we detect?

Reply:
We can detect the changes in error reporting, but only for unmarshaling (os.Read error is not covered).